2.7/s2i/bin/assemble

```diff
@@ -11,7 +11,7 @@ function should_collectstatic() {
 function virtualenv_bin() {
   if head "/etc/redhat-release" | grep -q "^CentOS Linux release 7" || \
      head "/etc/redhat-release" | grep -q "^Red Hat Enterprise Linux\( Server\)\? release 7" || \
-     head "/etc/redhat-release" | grep -q "^Fedora release 29 \(Twenty Nine\)\?"; then
+     head "/etc/redhat-release" | grep -q "^Fedora release 31 \(Thirty One\)\?"; then
```

Comment:
Can we leave just `grep -q "^Fedora release";` or are there still some non-EOL Fedora releases that use `virtualenv-${PYTHON_VERSION}`?

Reply:
You are right. We have only one version of Fedora now.